server/views/pages/overview/partials/timelineTab/_timelineEvent-ACTION_PLAN_CREATED.test.ts

```diff
@@ -42,32 +39,4 @@ describe('_timelineEvent-ACTION_PLAN_CREATED', () => {
       `View Jimmy Lightfingers's learning and work progress`,
     )
   })
-
-  it('should display ACTION_PLAN_CREATED timeline event given prison name not looked up', () => {
```

Comment:
This test, and all the others like it, are replaced by one in the mapper which ensures a prisonName is always populated with the name or falling back to the id